packages/desktop-client/src/components/reports/graphs/SankeyGraph.tsx

```diff
@@ -9,20 +9,13 @@ import {
 } from 'recharts';
 
 import Container from '../Container';
+import numberFormatterTooltip from '../numberFormatter';
 
 type SankeyProps = {
   style;
   data;
   compact: boolean;
 };
-type PotentialNumber = number | string | undefined | null;
-
-const numberFormatterTooltip = (value: PotentialNumber): number | null => {
-  if (typeof value === 'number') {
-    return Math.round(value);
-  }
-  return null; // or some default value for other cases
-};
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏thanks for extracting it into a reusable util!